backend/contracts/OasisReward.sol

```diff
@@ -56,7 +59,7 @@ contract OasisReward is ERC721Enumerable {
      * @param to address to mint the NFT to
      * @param base64EncodedSVG base64 encoded SVG image
      */
-    // Function to mint new tokens
+    /// Function to mint new tokens
```

Comment:
Redundant?

Reply:
Yes, removed this comment in PR since it's already contained in the above natspec documentation.